src/sentry/api/endpoints/organization_release_details.py

```diff
@@ -354,7 +407,10 @@ def get(self, request, organization, version):
                             sort=sort,
                             status_filter=status_filter,
                             query=query,
-                        )
+                        ),
+                        **self.get_first_and_last_releases(
+                            org=organization, filter_params=filter_params, sort=sort
```

Comment:
Wondering if we should just pass in `project_id` directly, since we already have it?

Reply:
Yeah you are right. It also does not make sense to send the entire `filter_params` object when I only need `project_id` and `environment`